port/scorecard/schema.go

```diff
@@ -183,8 +183,8 @@ resource "port_scorecard" "readiness" {
             operator = "isNotEmpty"
           }),
           jsonencode({
-            property = "author",
-            operator : "=",
```

Comment:
why you removed it?

Reply:
will it still work?

Reply:
Yeah it will work, it was the only one with the odd structure, the other json types are not like that